components/DataKeysTable/DataKeysTable.tsx

```diff
@@ -72,40 +74,54 @@ const DataKeysTable: React.FC<Props> = ({ address, isErc725Y }) => {
     return <p>⬆️ enter the address of your UP</p>;
   }
 
+  const findLinkForSchemaName = (schemaName) => {
+    const linkObj = SchemaLinks.find(
+      (linkItem) => linkItem.name === schemaName,
+    );
+    return linkObj ? linkObj.link : '#';
+  };
+
   return (
     <div className="columns is-multiline">
       {data.map((data) => {
+        const schemaLink = findLinkForSchemaName(data.schema.name);
         return (
-          <div className="column is-full" key={data.key}>
+          <div className="column is-full mt-4 dataKeyBox" key={data.key}>
             <div className="content py-5">
-              <h4 className="title is-4">
-                {data.schema.name}{' '}
-                <span className="tag is-medium mu-2 mb-2 mr-2 is-success">
+              <div className="title is-4 home-link">
+                <a href={schemaLink} target="_blank" rel="noopener noreferrer">
+                  {data.schema.name} ↗️
+                </a>
+                <span className="tag is-small mu-2 mb-2 mr-2 ml-2 is-info">
                   {data.schema.keyType}
                 </span>
-              </h4>
+              </div>
               <ul>
-                <li>
-                  Key: <code>{data.schema.key}</code>
+                <li className="mt-2">
+                  <strong>Key:</strong> <code>{data.schema.key}</code>
                 </li>
-                <li>
-                  Raw value{' '}
-                  <span className="tag is-medium mu-2 mb-2 mr-2 is-link is-light">
+                <li className="mt-4">
+                  <strong>Raw value: </strong>
+                  <span className="tag is-small mu-2 mb-2 mr-2 ml-2 is-link is-light">
```

Comment:
I'm not sure where `mu-2` comes from - i think we should rmeove it

You can simplify with: my-2 / mx-2 or m-2:
 - https://bulma.io/documentation/helpers/spacing-helpers/

Also, there is an issue here, this element should not be responsible for pushing these margins

<img width="599" alt="image" src="https://github.com/lukso-network/tools-erc725-inspect/assets/477945/a6bab9a9-0369-42d5-a360-eae0ac038b17">

Prefer doing this on the blocks elements rather than from the inline elements